src/Microsoft.DotNet.Darc/Darc/Helpers/RemoteFactory.cs

```diff
@@ -12,20 +12,20 @@ namespace Microsoft.DotNet.Darc.Helpers;
 
 internal class RemoteFactory : IRemoteFactory
 {
-    private readonly ICommandLineOptions _options;
+    private readonly CommandLineOptions _options;
```

Comment:
Curious - why did we drop the interface?

Reply:
We were just injecting CommandLindOptions, not ICommandLineOptions, I'll redo this part I think